ext/standard/basic_functions.stub.php

```diff
@@ -600,50 +600,50 @@ function str_starts_with(string $haystack, string $needle): bool {}
 
 function str_ends_with(string $haystack, string $needle): bool {}
 
-function chunk_split(string $str, int $chunklen = 76, string $ending = "\r\n"): string {}
+function chunk_split(string $string, int $length = 76, string $separator = "\r\n"): string {}
 
-function substr(string $str, int $start, ?int $length = null): string {}
+function substr(string $string, int $start, ?int $length = null): string {}
 
-function substr_replace(array|string $str, array|string $replace, array|int $start, array|int|null $length = null): string|array {}
+function substr_replace(array|string $string, array|string $replace, array|int $start, array|int|null $length = null): string|array {}
 
-function quotemeta(string $str): string {}
+function quotemeta(string $string): string {}
 
 function ord(string $character): int {}
 
 function chr(int $codepoint): string {}
 
-function ucfirst(string $str): string {}
+function ucfirst(string $string): string {}
 
-function lcfirst(string $str): string {}
+function lcfirst(string $string): string {}
 
-function ucwords(string $str, string $delimiters = " \t\r\n\f\v"): string {}
+function ucwords(string $string, string $delimiters = " \t\r\n\f\v"): string {}
 
-function strtr(string $str, string|array $from, ?string $to = null): string {}
+function strtr(string $string, string|array $from, ?string $to = null): string {}
 
-function strrev(string $str): string {}
+function strrev(string $string): string {}
 
 /** @param float $percent */
-function similar_text(string $str1, string $str2, &$percent = null): int {}
+function similar_text(string $string1, string $string2, &$percent = null): int {}
 
-function addcslashes(string $str, string $charlist): string {}
+function addcslashes(string $string, string $characters): string {}
 
-function addslashes(string $str): string {}
+function addslashes(string $string): string {}
 
-function stripcslashes(string $str): string {}
+function stripcslashes(string $string): string {}
 
-function stripslashes(string $str): string {}
+function stripslashes(string $string): string {}
 
-/** @param int $replace_count */
-function str_replace(array|string $search, array|string $replace, string|array $subject, &$replace_count = null): string|array {}
+/** @param int $count */
+function str_replace(array|string $search, array|string $replace, string|array $subject, &$count = null): string|array {}
 
-/** @param int $replace_count */
-function str_ireplace(array|string $search, array|string $replace, string|array $subject, &$replace_count = null): string|array {}
+/** @param int $count */
+function str_ireplace(array|string $search, array|string $replace, string|array $subject, &$count = null): string|array {}
 
-function hebrev(string $str, int $max_chars_per_line = 0): string {}
+function hebrev(string $string, int $max_chars_per_line = 0): string {}
```

Comment:
nit: in other places `character` is not abbreviated. I'm ok with `$max_chars_per_line` though, it already has very long a name.